ui/src/app/utils/runtime-utils.tsx

```diff
@@ -299,10 +299,10 @@ export const useRuntimeStatus = (currentWorkspaceNamespace): [
         return runtimeApi().deleteRuntime(currentWorkspaceNamespace);
       }],
       [RuntimeStatusRequest.Start, () => {
-        return leoRuntimesApi().startRuntime(currentWorkspaceNamespace, runtime.runtimeName);
+        return leoRuntimesApi().startRuntime(currentGoogleProject, runtime.runtimeName);
```

Comment:
This made me realize that it's kind of odd that we're starting runtimes directly w/ Leo but deleting runtimes through Workbench. As a result, this function needs to know how to use both APIs and take in both the workspaceNamespace and googleProject arguments.

Not something to change now but something to note for the future.

Reply:
Agreed, it's not a great state. Unfortunately we are a bit forced in both directions currently:

- setCookie must happen from the client (I'm fairly confident); this is a direct Leo API call
- createRuntime _should_ happen through the server to allow for handling of AoU-specific initialization logic; possibly this could be moved to the client-side but it would probably be really complicated

So now that the two connections exist, we can decide whether to proxy all calls through our API server (even the pass-throughs like start/stop) or just to use the existing direct connection from the client where possible. I could see an argument to proxy the calls instead for consistency, though this does add unnecessary endpoints / boilerplate. Possibly there's a better answer that comes with a longer term Terra convergence story